src/services/emailService.test.js

```diff
@@ -0,0 +1,30 @@
+import emailService from "./EmailService";
+import isValidEmail from "../components/helpers/validators/email-address-validator";
```

Comment:
why you check here method from diff file?

Reply:
fixed